ssd1306/src/main/java/com/google/android/things/contrib/driver/ssd1306/Ssd1306.java

```diff
@@ -238,6 +239,19 @@ public void setPixel(int x, int y, boolean on) throws IllegalArgumentException {
     }
 
     /**
+     * Sets the contrast for the display.
+     *
+     * @param level The contrast level (0-255).
+     */
+    public void setContrast(int level) throws IOException, IllegalArgumentException {
+        if (level < 0 || level > 255) {
+            throw new IllegalArgumentException("contrast out of bound:" + level);
```

Comment:
Let's put the acceptable range in the exception message, e.g. "Contrast level must be between 0 and 255"